packages/utils/merge.ts

```diff
@@ -0,0 +1,14 @@
+export default function objectAssign(target, ...agrvs) {
```

Comment:
I wonder why object spread cannot work.

Reply:
only the value not equal to `undefined`, will be assign. so...